cli/integration_tests/dry_json/single_package_no_config.t

```diff
@@ -39,7 +39,9 @@ Setup
         "hashOfExternalDependencies": "",
         "cache": {
           "local": false,
-          "remote": false
+          "remote": false,
+          "status": "MISS",
```

Comment:
This is a bit weird. I really don't want to customize for dry runs, but it also doesn't make too much sense to see "HIT" or "MISS" here since it didn't run. Should it say "WILL HIT" and "WILL MISS" instead?

Reply:
or maybe it's fine, since local:false, remote:false also have the same problem

Reply:
yeah I think MISS or HIT is fine here. maybe 'outcome' is a better name than 'status'?

edit: I can see status has been around longer than this pr. ignore.

Reply:
My two cents is that it's having `HIT`/`MISS` is fine as I feel like the "will be a" is implied by the dry run.